src/zoid/buttons/component.jsx

```diff
@@ -952,6 +953,13 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         value: () => sessionState,
       },
 
+      shopperInsightsUsed: {
```

Comment:
```suggestion
      getShopperInsightsUsed: {
```

we should probably call this `get...` because its a function not a property

Reply:
great callout here!